### PR TITLE
PostgreDriver: fix indexes reflection for indexes on expressions

### PR DESCRIPTION
**Pull request by jkuchar**
- bug fix
- BC break? no

Fix reflection for indexes on expressions. Columns are then not known then.

### CODE REVIEW

**Comment by dg** [comment]
Please fix coding style

**Comment by jkuchar** [comment]
fixed

File: src/Dibi/Drivers/PostgreReflector.php [hunk 1]
```diff
@@ -176,7 +176,11 @@ public function getIndexes(string $table): array
 			$indexes[$row['relname']]['name'] = $row['relname'];
 			$indexes[$row['relname']]['unique'] = $row['indisunique'] === 't' || $row['indisunique'] === true;
 			$indexes[$row['relname']]['primary'] = $row['indisprimary'] === 't' || $row['indisprimary'] === true;
+			$indexes[$row['relname']]['columns'] = [];
 			foreach (explode(' ', $row['indkey']) as $index) {
+				if (!isset($columns[$index])) {
```
Comment (dg) [inline src/Dibi/Drivers/PostgreReflector.php]: Why not 
```
if (isset($columns[$index])) {
	$indexes[$row['relname']]['columns'][] = $columns[$index];														}
```

Comment (jkuchar) [reply src/Dibi/Drivers/PostgreReflector.php]: Thanks. 👍

**Pull request merged**